lib/util/util.js

```diff
@@ -119,40 +119,40 @@ const Util = {
      *
      * @param {object} properties javascript object in .mcdevrc.json
      * @param {boolean} [silent] set to true for internal use w/o cli output
-     * @returns {boolean|String[]} file structure ok OR list of fields to be fixed
+     * @returns {Promise<boolean|String[]>} file structure ok OR list of fields to be fixed
      */
-    checkProperties: function (properties, silent) {
-        if (!fs.existsSync(Util.configFileName) || !properties) {
+    checkProperties: async function (properties, silent) {
+        if (!(await fs.pathExists(Util.configFileName)) || !properties) {
```

Comment:
why switch from exists to pathExists?
i got the sync/async part but that's a separate thing here

Reply:
https://stackoverflow.com/questions/56488179/what-is-the-difference-between-fsextra-pathexists-and-fsextra-exists